### ISSUE TITLE
[gatsby-source-contentful] Image transforms still have default width

### ISSUE THREAD

**Issue by LuudJanssen** [gatsbyjs/gatsby#14156]
## Description

[This pull request](https://github.com/gatsbyjs/gatsby/pull/7680) removed the default image width for image transforms to enable you to only set a `height` and have the image infer the correct width automatically, just like it does with the height when setting a `width`.

This default still exists and can be removed in gatsby-source-contentful because [this pull request](https://github.com/gatsbyjs/gatsby/pull/6970) for gatsby-plugin-sharp infers the defaults automatically if neither width nor height are given.

### Steps to reproduce

Source an image from contentful and only set the height for the image transform with a query like:

```
query getImageFixedHeight {
  allContentfulImages {
    nodes {
      image {
        fixed(height: 160) {
          width
          height
        }
      }
    }
  }
}
```

### Expected result

`height` should be 160 and `width` should be scaled according to the aspect ratio of the image.

### Actual result

`width` is always 400.

### Environment
  System:
    OS: Windows 10
    CPU: (8) x64 Intel(R) Core(TM) i7-4700MQ CPU @ 2.40GHz
  Binaries:
    Yarn: 1.13.0 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 6.9.0 - C:\Program Files\nodejs\npm.CMD
  Languages:
    Python: 2.7.15 - /c/Python27/python
  Browsers:
    Edge: 42.17134.1.0
  npmPackages:
    gatsby: ^2.4.3 => 2.4.3 
    gatsby-cli: ^2.5.8 => 2.5.13 
    gatsby-image: ^2.0.41 => 2.0.41 
    gatsby-plugin-material-ui: ^2.0.0-alpha.4 => 2.0.0-beta.0 
    gatsby-plugin-react-helmet: ^3.0.12 => 3.0.12 
    gatsby-plugin-sharp: ^2.0.37 => 2.0.37 
    gatsby-plugin-styled-components: ^3.0.7 => 3.0.7 
    gatsby-source-contentful: ^2.0.40 => 2.0.56 
    gatsby-transformer-sharp: ^2.1.19 => 2.1.19

**Comment by LuudJanssen** [comment]
I'm working on a pull request.

**Comment by sidharthachatterjee** [comment]
Published in `gatsby-source-contentful@2.0.69`
